vizro-core/docs/pages/user-guides/table.md

```diff
@@ -1,25 +1,315 @@
+                    title="Modified Dash AG Grid",
+                    figure=dash_ag_grid(
+                        data_frame=df,
+                        columnDefs=columnDefs,
+                        defaultColDef={"resizable": False, "filter": False, "editable": True},
+                    ),
+                )
+            ],
+        )
+
+        dashboard = vm.Dashboard(pages=[page])
+
+        Vizro().build(dashboard).run()
+        ```
+    === "app.yaml"
+        ```yaml
+        # Still requires a .py to register data connector in Data Manager and parse yaml configuration
+        # See from_yaml example
+        pages:
+          - components:
+              - figure:
+                  _target_: dash_ag_grid
+                  data_frame: gapminder
+                  columnDefs:
+                    - field: country
+                    - field: continent
+                    - field: year
+                    - field: lifeExp
+                      valueFormatter:
+                        function: "d3.format('.1f')(params.value)"
+                    - field: gdpPercap
+                      valueFormatter:
+                        function: "d3.format('$,.1f')(params.value)"
+                      cellStyle:
+                        styleConditions:
+                          - condition: params.value < 1045
+                            style:
+                              backgroundColor: "#ff9222"
+                          - condition: params.value >= 1045 && params.value <= 4095
+                            style:
+                              backgroundColor: "#de9e75"
+                          - condition: params.value > 4095 && params.value <= 12695
+                            style:
+                              backgroundColor: "#aaa9ba"
+                          - condition: params.value > 12695
+                            style:
+                              backgroundColor: "#00b4ff"
+                    - field: pop
+                      type: rightAligned
+                      valueFormatter:
+                        function: "d3.format(',.0f')(params.value)"
+                  defaultColDef:
+                    resizable: false
+                    filter: false
+                    editable: true
+                title: Dash AG Grid
+                type: ag_grid
+            title: Example of a Dash AG Grid
+        ```
+    === "Result"
+        [![Table2]][Table2]
+
+    [Table2]: ../../assets/user_guides/table/styled_table.png
+
+If the available arguments are not sufficient, there is always the option to create a [custom AG Grid callable](custom_tables.md).
+
+## Dash DataTable
+
+Similar to AG Grid, the [Dash DataTable](https://dash.plotly.com/datatable) is an interactive table/grid component designed for viewing, editing, and exploring large datasets.
+
+The Vizro [`Table`][vizro.models.Table] model is based on the [Dash DataTable](https://dash.plotly.com/datatable).
+
```

Comment:
```suggestion
In general, we recommend using [AG Grid](#ag-grid) for tables unless you have a particular reason to prefer Dash DataTable.
```

It's worth repeating this somewhere at the beginning of the section I think. Maybe in one of those callouts? Or just in plain text, up to you.

Reply:
Ok I added it as plain text, I think we are clear enough. I mean it's not like the datatable is broken haha